languages/java/exercises/concept/classes/.docs/after.md

```diff
@@ -0,0 +1,101 @@
+The primary object-oriented construct in Java is the _class_, which is a combination of data ([_fields_][fields]), also known as instance variables, and behavior ([_methods_][methods]). The fields and methods of a class are known as its _members_.
+
+Access to members can be controlled through access modifiers, the two most common ones being:
+
+- [`public`][public]: the member can be accessed by any code (no restrictions).
+- [`private`][private]: the member can only be accessed by code in the same class.
+
+It is customary to specify an access modifier for all members. If no access modifier is specified, it will default to `private`.
+
+The above-mentioned grouping of related data and behavior plus restricting access to members is known as [_encapsulation_][encapsulation], which is one of the core object-oriented concepts.
+
+You can think of a class as a template for creating instances of that class. To [create an instance of a class][creating-objects] (also known as an _object_), the [`new` keyword][new] is used:
+
+```csharp
+class Car
+{
```

Comment:
normally in Java left curly bracket is on the same line of the class name
so 
```
class Car {
}
```
This is valid for all the code in this page
This is valid for block of code too